lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -1,6 +1,17 @@
 # frozen_string_literal: true
 
 module Kramdown
+  require "kramdown/converter/html"
+
+  class JekyllKD2HTML < Converter::Html
```

Comment:
What do you think about renaming this to `JekyllHtmlConverter` to keep with a similar naming convention as the other class in this file?

Reply:
While I did have that exact name (actually it was `JekyllHTMLConverter`) for the class when starting with the patch, the segment `HtmlConverter` felt ambiguous to me. Would someone interpret it as a class for *converting HTML* (like how we have the `sass-converter` plugin).

Regardless, I am open to renaming the class.

Reply:
+1 on keeping it clear. My only nit here is that "KD2" is not super clear. We can think of the entire class name with the `Kramdown::` prefix for clarity if you think that would allow the inner class to be less wordy, but I don't mind long class names.